apps/standalone/src/app/hooks/useCliArtifacts.ts

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { fetchCliArtifacts } from '../utils/apiCalls';
+
+import { CliArtifactsResponse } from '@flightctl/ui-components/src/types/extraTypes';
+
+export const useCliArtifacts = () => {
```

Comment:
IMO we dont need this hook, lets remove it completely

Reply:
Done, as well as the other suggested changes.